doc/api/deprecations.md

```diff
@@ -864,3 +864,15 @@ details.
 [from_arraybuffer]: buffer.html#buffer_class_method_buffer_from_arraybuffer_byteoffset_length
 [from_string_encoding]: buffer.html#buffer_class_method_buffer_from_string_encoding
 [`REPLServer.clearBufferedCommand()`]: repl.html#repl_replserver_clearbufferedcommand
+
+<a id="DEP0087"></a>
+### DEP0087: node::Init() and node::Start() that take argc/argv.
+
+Type: Runtime
```

Comment:
Now that you removed `NODE_DEPRECATED`, it's a "Documentation-only" deprecation, not a "Runtime" one.